packages/components/psammead-sitewide-links/src/index.jsx

```diff
@@ -44,10 +48,16 @@ const StyledParagraph = styled.p`
   padding: ${GEL_SPACING_DBL} 0;
 `;
 
-const SitewideLinks = ({ links, copyrightText, externalLink, service }) => (
+const SitewideLinks = ({
+  links,
+  TrustProjectLink,
+  copyrightText,
+  externalLink,
+  service,
+}) => (
   <SitewideLinksWrapper service={service}>
-    <ConstrainedWrapper>
-      <List links={links} />
+    <ConstrainedWrapper isTrustProject={typeof TrustProjectLink}>
+      <List links={links} TrustProjectLink={TrustProjectLink} />
```

Comment:
we should probably stick to using camelCase which means that `TrustProjectLink` should be updated to `trustProjectLink` everywhere it's mentioned